eth_abi/utils/padding.py

```diff
@@ -4,23 +4,23 @@
 
 
 @curry
-def zpad(value, length):
+def zpad(value:str, length:int) -> str:
```

Comment:
Can you add spaces after the `:` for style consistency.

Reply:
Oh sure. I was going to lint it after I was done with it. I will do that before asking for a review its still WIP. :smile: